schemas/content/product.schema.json

```diff
@@ -155,16 +135,6 @@
           "type": "string",
           "description":
             "The URL for the primary Product View of the product variant page."
-        },
-        "xdm:manufacturerName": {
```

Comment:
This proposal is being far to open, we know that many customers have to represent simular concepts for attributes and variants. But we are not giving them guidance on how to be consistent in the schema. This must be improved.